dev_docs/kibana_platform_plugin_intro.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: kibPlatformIntro
+slug: /kibana-dev-docs/platform-intro
+title: What is the Kibana platform?
+summary: An introduction to what the Kibana platform is and what it means to build a plugin on top of it.
+date: 2021-01-06
+tags: ['kibana','onboarding', 'dev', 'architecture']
+---
+
+From an end user perspective, Kibana is a tool for interacting with Elasticsearch, providing an easy way 
+to visualize and analyze data.
+
+From a developer perspective, Kibana is a platform that provides a set of tools to build not only the UI you see in Kibana today, but
+almost any UI that can be dreamt up. The platform provides developers the ability to build applications, or inject extra functionality into 
+already existing applications. Did you know that everything you see in the
+Kibana UI is built inside a plugin? If you removed all plugins from Kibana, you'd be left without any UI, just a set of developer tools!
```

Comment:
nit: the navigation UI is actually part of Core, so if you removed all apps, I think you'd be left with an empty navbar.